contribute/styleguides/gettext/index.md

```diff
@@ -7,7 +7,9 @@ Idea is not to update fields that do not add any value and fields that are not r
 
   * The `.mo` files should be compiled by the packaging system and locally during development.
 
-* If no reason not to, use English string as message id (i.e. default string) and make translations for other languages (e.g. Finnish).
+* If no reason not to, use English string as message id (i.e. default string) and make translations for other languages (e.g. Finnish).  
```

Comment:
I would organize this bullet differently, because this is not anymore the case with main aplus code. Would this be clearer for the reader (as an idea, you can choose the wordings as you wish):
* In main a-plus code msgID format is used to identify text strings and separate translation files also for English
* In other repositories, the English text version is used as message identifier for translations.

Reply:
Good idea, I'll fix this!